src/MapPanel.jsx

```diff
@@ -22,6 +22,7 @@ const WARNING_MESSAGE = 'Due to your filter selections, the “Complaints per' +
 
 export class MapPanel extends React.Component {
   render() {
+    /* eslint complexity: ["error", 5] */
```

Comment:
These types of comment apply to the whole module, no the next line... so I just move them to the top of the file

`// eslint-disable-next-line complexity`

is what to use for a single line function that is ridiculous